docs/01-introduction.Rmd

```diff
@@ -183,7 +183,19 @@ Note that [R will silently ignore the last line of your `.Rprofile`](https://yih
 
 ## R Markdown vs. Markdown {#output-format}
 
-If you are not familiar with R Markdown\index{R Markdown}, please see Appendix \@ref(r-markdown) for a quick tutorial. When you create a new post, you have to decide whether you want to use R Markdown or plain Markdown\index{Markdown}, as you can see from Figure \@ref(fig:new-post). The main differences are:
+If you are not familiar with R Markdown\index{R Markdown}, please see Appendix \@ref(r-markdown) for a quick tutorial. When you create a new post, you have to decide whether you want to use R Markdown or plain Markdown\index{Markdown}, as you can see from Figure \@ref(fig:new-post). 
+
+The following table summarises the differences between the three options, followed by more detailed explanations below.
```

Comment:
`The following table` -> `Table \@ref(tab:md-diff)`.

`summarise` -> `summarize`

delete `more` before `detailed`